src/GridFieldAddNewInlineButton.php

```diff
@@ -86,11 +86,11 @@ public function getHTMLFragments($grid)
 
         $fragment = $this->getFragment();
 
-        if (!$editable = $grid->getConfig()->getComponentByType('GridFieldEditableColumns')) {
+        if (!$editable = $grid->getConfig()->getComponentByType(GridFieldEditableColumns::class)) {
             throw new Exception('Inline adding requires the editable columns component');
         }
 
-        Requirements::javascript(THIRDPARTY_DIR . '/javascript-templates/tmpl.js');
+        Requirements::javascript('gridfieldextensions/javascript/tmpl.js');
```

Comment:
My example should work see https://github.com/silverstripe/silverstripe-admin/pull/121/files for other usages